.github/workflows/ci.yml

```diff
@@ -52,6 +52,15 @@ on:
         required: false
         type: boolean
         default: true
+      fix-release:
```

Comment:
Maybe use rebuild rather than fix consistently? Not really sure what fix means.

Reply:
Changed to "do-release"